#!/usr/bin/python3

def raise_exception_msg(message=""):
    rase NameError(message)
